app/scripts/models/account.js

```diff
@@ -414,10 +416,13 @@ define(function (require, exports, module) {
         }
       })
       .then((updatedSessionData) => {
+        this._metrics.logEvent('core.signin.success');
```

Comment:
We have so many of these `signin.success` now, I don't even know which one is anymore. Maybe we should take some out / clean up?